board/solidrun/mx6cuboxi/mx6cuboxi.c

```diff
@@ -875,4 +877,113 @@ void board_init_f(ulong dummy)
 	/* load/boot image from boot device */
 	board_init_r(NULL, 0);
 }
+
+#ifdef CONFIG_SPL_BOARD_PROVISION
+
+struct srkfdt {
+	uint32_t srk0;
+	uint32_t srk1;
+	uint32_t srk2;
+	uint32_t srk3;
+	uint32_t srk4;
+	uint32_t srk5;
+	uint32_t srk6;
+	uint32_t srk7;
+};
+
+void spl_board_provision() {
+	u32 val, i;
+	u32 mac0, mac1;
+	char c;
+	char input[100];
+
+	hab_rvt_report_status_t *hab_rvt_report_status;
+	hab_rvt_report_status = (hab_rvt_report_status_t *)HAB_RVT_REPORT_STATUS;
+	enum hab_status status = 0;
+	enum hab_config config = 0;
+	enum hab_state state = 0;
+	status = hab_rvt_report_status(&config, &state);
```

Comment:
What is status for if it's not used?

Reply:
Removed status, it was for my own education about HAB, but checking it isn't required.